ui/src/institutions/components/GridLink.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import ExternalLink from '../../common/components/ExternalLink';
+
+function GridLink({ grid }) {
+  const href = `//grid.ac/institutes/${grid}`;
+  return (
+    <div>
```

Comment:
Let's make it `span`, so that we can also display inline when need to, and let parent decide on the layout.